### DISCUSSION TITLE
Why remix is not loading binance chain ???

### DISCUSSION THREAD

**Discussion by ErMuSaM1** [remix-run/remix#7921] [Q&A] [answered]
Its been 4 days am trying to deploy a contract in binance chain using remix. But remix failed to load binance chain with metamask extension on firefox...

i tried with trustwallet and atomic wallet also. But it fails every time...

whats going on... tell me how to work it out

**Comment by MichaelDeBoey** [comment, marked as answer]
Hello 👋
I think you may be in the wrong place. This GitHub repo is all about the Remix Web Framework which you can learn about at https://remix.run.
Closing this as you have mixed this up with the Remix IDE (https://remix-project.org) which is a completely different project.
